lib/widgets/loading_indicator.dart

```diff
@@ -0,0 +1,10 @@
+import 'package:flutter/material.dart';
+
+class LoadingIndicator extends StatelessWidget {
```

Comment:
suggestion: Since this only wraps `CircularProgressIndicator` in a `Center`, a more descriptive name could be `CenteredCircularProgressIndicator`.